### PR TITLE
Update description

### PR DESCRIPTION
**Pull request by jopit**
## What type of PR is this ? 

- [ ] Bug fix
- [x] Enhancement

## What does this PR do ?

Enhances the plugin's description.

## Which issue(s) does this PR fix ?

https://github.com/eclipse/codewind/issues/2462

#### Link to the [Codewind repository](https://github.com/eclipse/codewind/issues) issue(s) this PR fixes or references:

https://github.com/eclipse/codewind/issues/2462

## Does this PR require a documentation change ?

No

### CODE REVIEW

File: dev/src/main/resources/META-INF/plugin.xml [hunk 1]
```diff
@@ -14,8 +14,50 @@
...
 
     <description><![CDATA[
-        Adds support for developing cloud-native, containerized web applications.
-    ]]>    </description>
+      <h2>Codewind</h2>
+      <p>
+        Adds tools to help build cloud-native, containerized Java
+        applications. Create an application from a template project
+        and launch, update, test, and debug in containers on the
+        desktop. Supports Codewind version 0.10.0. For more
```
Comment (sishida) [inline dev/src/main/resources/META-INF/plugin.xml]: Maybe say, "0.10.0 and onward"?

File: dev/src/main/resources/META-INF/plugin.xml [hunk 2]
```diff
@@ -14,8 +14,50 @@
...
+        information, see <a href="https://www.eclipse.org/codewind/">https://www.eclipse.org/codewind/</a>.
```
Comment (sishida) [inline dev/src/main/resources/META-INF/plugin.xml]: Oh, we can add hyperlinks? In that case, maybe say, `...see the <a href="https://www.eclipse.org/codewind/">Codewind website</a>` instead?

File: dev/src/main/resources/META-INF/plugin.xml [hunk 3]
```diff
@@ -14,8 +14,50 @@
...
+        <li>Create new containerized projects or add existing ones.</li>
+        <li>View your containerized projects, including the application and build status.</li>
+        <li>Access application, build, and container logs in the Output view.</li>
+        <li>Toggle the project auto build setting and manually initiate project builds.</li>
+        <li>Open your application in a browser.</li>
+      </ul>
+      <h2>Getting started with Codewind</h2>
+      <p>
+        For more information, see
+        <a href="https://www.eclipse.org/codewind/intellij-getting-started.html">https://www.eclipse.org/codewind/intellij-getting-started.html</a>.
```
Comment (sishida) [inline dev/src/main/resources/META-INF/plugin.xml]: Hyperlink with title of the file (minus "Tech Preview"): `<a href="https://www.eclipse.org/codewind/intellij-getting-started.html">Installing Codewind for IntelliJ</a>`

File: dev/src/main/resources/META-INF/plugin.xml [hunk 4]
```diff
@@ -14,8 +14,50 @@
...
+          Submitting issues:
+          <a href="https://github.com/eclipse/codewind/issues">https://github.com/eclipse/codewind/issues</a>.
+        </li>
+        <li>
+          Contributing:
+          <a href="https://github.com/eclipse/codewind-intellij/blob/master/CONTRIBUTING.md">https://github.com/eclipse/codewind-intellij/blob/master/CONTRIBUTING.md</a>.</li>
+      </ul>
+      <h2>Developing</h2>
+      <p>To develop and debug Codewind for IntelliJ, see
+        <a href="https://github.com/eclipse/codewind-intellij#developing">https://github.com/eclipse/codewind-intellij#developing</a>.
```
Comment (sishida) [inline dev/src/main/resources/META-INF/plugin.xml]: What about...`<a href="https://github.com/eclipse/codewind-intellij#developing">Developing</a>`?

File: dev/src/main/resources/META-INF/plugin.xml (hunk 1 above)
Comment (jopit) [reply dev/src/main/resources/META-INF/plugin.xml]: At this point the 0.10.0 version of the plugin only supports the 0.10.0 version of Codewind (this is true for all the IDEs). Until that changes, I think we'll have to update the supported version every time we release.

File: dev/src/main/resources/META-INF/plugin.xml (hunk 3 above)
Comment (jopit) [reply dev/src/main/resources/META-INF/plugin.xml]: Changed

File: dev/src/main/resources/META-INF/plugin.xml (hunk 4 above)
Comment (jopit) [reply dev/src/main/resources/META-INF/plugin.xml]: Changed

File: dev/src/main/resources/META-INF/plugin.xml (hunk 2 above)
Comment (jopit) [reply dev/src/main/resources/META-INF/plugin.xml]: Changed

**Comment by jopit** [comment]
@sishida Hi Sarah, I've made changes based on you review, could you please re-review?

File: dev/src/main/resources/META-INF/plugin.xml (hunk 1 above)
Comment (sishida) [reply dev/src/main/resources/META-INF/plugin.xml]: Oh, OK, so this text is appearing specifically for the 0.10.0 plugin. I had thought it was appearing across any and all versions. If the text is version-specific, I could see why we'd want to leave the text as it is with mention of 0.10.0. Thanks!

**Pull request merged**